reasoning.md

```diff
@@ -24,3 +24,15 @@ We believe active membership is important as it brings a host of other benefits
 Active membership entails voting (democracy) and meetings as a base, but importantly involves recognized volunteering work that helps the consortium function and thus spreads the administrative work to all. An important reason this should be recognized is that it brings about transparency that motivates others to chip in as well. A financial stake of recurring payment is also a form of active membership, and in order for it not to be excluding, as the financial situation can vary a lot between the different members, the amount is kept flexible. Lastly, face-to-face meetings are an important part of being an active member as there is little substitute for physical gatherings.
 
 One important side-effect of being an active member is that it fosters trust through active collaboration. Both for the parties involved in the collaboration but also for other members trusting in a good outcome. Furthermore, face-to-face gatherings over a weekend allows members to see each others as humans, laying the foundation for a culture of care. When nurtured this should expand into positive feedback loops that strengthens the consortium.
+
+## Handling Conflicts
+
+As much as possible, we try to view conflicts as opportunities for growth, both in members and for the consortium: members have an opportunity to better understand the values and experiences they bring with them; the consortium gets an opportunity to revise its practices and documents. We therefore encourage members involved in a conflict to reflect and articulate their underlying personal needs and previous experiences that support their viewpoint. A clear understanding of both needs and past experiences, from all sides of a conflict, can make it easier to resolve, with, for example, voluntary behaviour changes from parties or revised practices in the consortium. This in turn leads to an increase in the level of trust and collaboration betwen members. This is the best case and preferred scenario.
+
+Sometimes, even with the best intentions from all sides, conflicts may be recurrent because, for example, of persisting personal habits or communication patterns. In this case, vouching members can remove their individual vouches for the other member(s) they perceive as problematic. This provides concrete consequences for problematic behaviour, providing an additional incentive for change.
+
+It is sometimes however not always possible to resolve all conflicts, especially when they are related to contradicting long-term visions or deeply-held values. In the cases where these tensions cannot cohabit in the same consortium, it is best for the different sides to recognize the situation and agree to an amicable split. If that cannot be achieved amicably, the largest group of the consortium can stop vouching for the smaller one (possibly comprised of a single individual), effectively removing them.
+
+## References
```

Comment:
Are these references for the document in general or this section specifically? If the latter, please clarify this.

Reply:
The references are intended for the whole document (so feel free to add yours!). To be more specific, I could add in the text from which part of the book I got some of the ideas. 

I would be personally fine with this but the document might start having an academic feeling that could put off some people.

Reply:
Ok :+1: Maybe rename the section to Inspiration or something like that? (ok I'm nitpicking now, feel free to click resolve)